docs/docs-content/enterprise-version/upgrade/upgrade-k8s/airgap.md

```diff
@@ -0,0 +1,432 @@
+    NAME: cert-manager
+    LAST DEPLOYED: Thu Feb 22 19:42:33 2024
+    NAMESPACE: default
+    STATUS: deployed
+    REVISION: 2
+    TEST SUITE: None
+    ```
+
+13. Prepare the Palette configuration file `values.yaml`. If you saved `values.yaml` used during the Palette
+    installation, you can reuse it for the upgrade. Alternatively, follow the
+    [Kubernetes Installation Instructions](../../install-palette/install-on-kubernetes/install.md) to populate your
+    `values.yaml`.
+
+:::warning
+
+Ensure that the `values.yaml` file is ready before proceeding. Specifically, make sure that the `ociPackEcrRegistry` and
+`ociImageRegistry` configurations include the parameters necessary to interact with your `spectro-images` and
+`spectro-packs` repositories.
+
+:::
+
+13. Upgrade the image-swap chart with the following command. Point to the `palette/values.yaml` file from step twelve.
+
+    ```shell
+    helm upgrade --values palette/values.yaml \
+    image-swap extras/image-swap/image-swap-*.tgz --install
+    ```
+
+    You should receive an output similar to the following.
+
+    ```shell
+    Release "image-swap" has been upgraded. Happy Helming!
+    NAME: image-swap
+    LAST DEPLOYED: Thu Feb 22 19:44:13 2024
+    NAMESPACE: default
+    STATUS: deployed
+    REVISION: 2
+    TEST SUITE: None
+    ```
+
+14. Upgrade the reach-system chart with the following command. Point to the `palette/values.yaml` file from step twelve.
+
+    ```shell
+    helm upgrade --values palette/values.yaml \
+    reach-system extras/reach-system/reach-system-\*.tgz --install
+    ```
+
+    You should receive an output similar to the following.
+
+    ```shell
+    Release "reach-system" has been upgraded. Happy Helming!
+    NAME: reach-system
+    LAST DEPLOYED: Thu Feb 22 19:47:10 2024
+    NAMESPACE: default
+    STATUS: deployed
+    REVISION: 2
+    TEST SUITE: None
+    ```
+
+15. Upgrade Palette with the following command.
+
+    ```shell
+    helm upgrade --values palette/values.yaml \
+    hubble palette/spectro-mgmt-plane-\*.tgz --install
+    ```
+
+    You should receive an output similar to the following.
+
+    ```shell
+    Release "hubble" has been upgraded. Happy Helming!
+    NAME: hubble
+    LAST DEPLOYED: Thu Feb 22 20:05:24 2024
+    NAMESPACE: default
+    STATUS: deployed
+    REVISION: 2
+    TEST SUITE: None
+    ```
+
+16. Use the following command to track the upgrade process.
+
+    ```shell
+    kubectl get pods --all-namespaces --watch
+    ```
+
+    :::tip
+
+    For a more user-friendly experience, consider using [K9s](https://k9scli.io/) or a similar tool to track the
+    upgrade.
+
+    :::
+
+    The upgrade usually takes up to five minutes. Palette is upgraded when the deployments in the namespaces
+    `cp-system`, `hubble-system`, `ingress-nginx`, `jet-system` , and `ui-system` are in the **Ready** status.
+
+## Validate
+
+1. In your terminal, issue the following command.
+
+   ```shell
+   helm ls
```

Comment:
The Helm list command is not a good indicator. A Helm chart could be deployed while still having pods that never come up or are crashing. Our best bet is to use the validation we have in the installation guide.